metric-anomaly-task-manager/src/main/resources/configs/common/application.conf

```diff
@@ -0,0 +1,7 @@
+service.name = metric-anomaly-rule-manager
+service.admin.port = 8099
+
+main.class = org.hypertrace.alerting.metric.alerting.rule.manager.AlertingRuleManager
```

Comment:
Let's change the repo name and package structure as follow-up PR.

Reply:
yeah